apps/directory/src/components/configuration/FilterConfigUI.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div>
+    <div class="mb-3">
+      <h3>Filters</h3>
+      <div class="d-flex justify-content-between flex-wrap">
+        <small class="d-inline-block"
+          >Rearrange the filters by dragging and dropping. Unchecking a filter
+          means it will be shown under <i>More filters</i> by default.</small
+        >
+        <button @click="emitAdd" class="btn btn-info mt-2">
+          Add new filter
+          <span class="fa fa-plus fa-lg ml-1" aria-hidden="true"></span>
+        </button>
+      </div>
+    </div>
+
+    <draggable
+      v-model="appConfig.filterFacets"
+      group="filterFacets"
+      @start="dragStart()"
+      @end="sync"
+      item-key="facetTitle"
+    >
+      <template #item="{ element }">
+        <div
+          @click="activeFilter = element.facetTitle"
+          class="list-group-item d-flex"
+          :class="{ editing: element.index === filterIndex }"
+        >
+          {{ element.index + 1 }}. {{ element.label || element.facetTitle }}
+          <small class="ml-auto" v-if="element.builtIn"
+            >Rearranging this has no effect in the application.</small
+          >
+          <label v-if="!element.builtIn" class="ml-auto"
+            ><input type="checkbox" @change="sync"
```

Comment:
I don't understand what i can do with the checkboxes

Reply:
but now the checkboxes are wired in :) that actually will do something now

Reply:
the show more feature was dropped in previous version. so renamed the text to just hide

Reply:
also fixed layout as requested